### PR TITLE
hanlder click item

### PR DESCRIPTION
**Pull request by tungct97**
#10 hanlder click item
#Redmine 
https://edu-redmine.framgia.vn/issues/3174

### CODE REVIEW

**Comment by tungct97** [comment]
ready

File: app/src/main/java/com/framgia/moneymanagement/data/source/remote/IncomeRemoteDataSource.java [hunk 1]
```diff
@@ -37,4 +37,17 @@ public void getIncomes(ValueEventListener valueEventListener) {
                 .child(Income.Key.INCOME)
                 .addValueEventListener(valueEventListener);
     }
+
+    @Override
+    public void deleteIncome(String id,
+                             OnCompleteListener onCompleteListener,
+                             OnFailureListener onFailureListener) {
+        mFirebaseDatabase.getReference(User.Key.USER).
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/moneymanagement/data/source/remote/IncomeRemoteDataSource.java]: dấu . nên ở dòng dưới

File: app/src/main/java/com/framgia/moneymanagement/screen/income/IncomeFragment.java [hunk 2]
```diff
@@ -73,4 +87,24 @@ public void onClick(View view) {
                 break;
         }
     }
+
+    @Override
+    public void OnLongClickListener(final String id) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/moneymanagement/screen/income/IncomeFragment.java]: new AlertDialog.Builder(getActivity())
.setMessage(R.string.title_delete)
.setPositiveButton(R.string.title_yes, new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialogInterface, int i) {
                mPresenter.deleteIncome(id);
             }
        });
.setNegativeButton(R.string.title_no, null)
.show();

**Comment by DoanVanToan** [comment]
commented

**Comment by tungct97** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**